src/menu/settings_menu_template.js

```diff
@@ -52,6 +53,25 @@ export const settingsMenu = {
         settings.set('startInTrayPref', startInTrayPref);
         item.checked = startInTrayPref;
       }
+    },
+    {
+      id: 'openAppAtLoginMenuItem',
+      label: 'Open At Login',
+      type: 'checkbox',
+      click: (item) => {
+        const openAppAtLoginPref = !settings.get('openAppAtLoginPref');
+        settings.set('openAppAtLoginPref', openAppAtLoginPref);
+        item.checked = openAppAtLoginPref;
+        
+        // Allows Windows & macOS to start the app at login
+        // Note: having the "startup apps" panel open breaks the functionality - Windows bug?
+        const exeName = path.basename(process.execPath);
+        app.setLoginItemSettings({
+        	openAtLogin: item.checked,
```

Comment:
Just for the sake of it being extremely obvious while reading the code, I'm a fan of this line saying:  
```js
openAtLogin: openAppAtLoginPref,
```  
But it doesn't affect functionality so it isn't that big of a deal at the end of the day.

Reply:
Ah, you're right! Good catch!

Reply:
Thanks for making the change, I appreciate it! :)